astra_mini.launch.py

```diff
@@ -0,0 +1,85 @@
+from launch import LaunchDescription
+from ament_index_python import get_package_share_directory
+from launch.actions import DeclareLaunchArgument, OpaqueFunction
+from launch_ros.actions import ComposableNodeContainer, Node, PushRosNamespace
+from launch_ros.descriptions import ComposableNode
+from launch.substitutions import EnvironmentVariable, LaunchConfiguration, PathJoinSubstitution
+
+
+def launch_setup(context, *args, **kwargs):
```

Comment:
I prefer to avoid the `OpacueFunction` . It can be done like this:
```python
    namespace_ext = PythonExpression(
        ["''", " if '", namespace, "' == '' or '", namespace, "' == '/'", "else ", "'/", namespace, "'"]
    )
    remapping.append(("/tf", LaunchConfiguration("tf_namespaced", default=[namespace_ext, "/tf"])

Reply:
For me `OpacueFunction` looks much better and is used in repo like `zed-ros2-wrapper`. `PythonExpression` is not readable.